DEV_ENVIRONMENT.md

```diff
@@ -0,0 +1,30 @@
+# How to set up your development environment
+
+1. Make sure you have the following tools installed on your computer: `Docker`, `node` >= 12, a JDK 11 and Maven. 
+We recommend to use [`nvm`](https://github.com/nvm-sh/nvm) to install and manage `node` versions and 
+[`jabba`](https://github.com/shyiko/jabba) to install and manage JDK versions, but this is optional.
+
+2. You can use any IDE you prefer, but we recommend on using one that supports our backend and frontend code (Java + Maven, TypeScript, SCSS)
+and ideally offers support for the Spring framework (backend) and Angular framework (frontend) as this really improves
+the developing experience.
+
+3. Fork the O-Neko project on GitHub and clone your O-Neko git repository.
+
+4. Open the project in your IDE or editor and navigate in a terminal to the `/frontend` directory and run `npm install`.
+This will install all frontend dependencies. You should do this regularly to always have the latest dependencies installed.
+The Java dependencies are managed by Maven and should be downloaded automatically, depending on your editor.
+
+5. O-Neko relies on a MongoDB. You can run it anywhere you want, but by default the application will try to connect to a database
+running on localhost with MongoDB's default port 27017. You can easily start that MongoDB by running `docker-compose up -d` in the
+project's root directory. If you want to use another MongoDB make sure to change the settings in the `/src/main/resources/application.yaml`.
```

Comment:
You might want to point out how the mongo-db-access can be changed through the application.yml's settings.

Reply:
I thought that one line in the `application.yaml` was so easy that it's not worth documenting. I could totally change that though, if you think it's necessary.